x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/fields_list_item.tsx

```diff
@@ -185,26 +185,17 @@ export const FieldsListItem = React.memo(function FieldListItemComponent({
                 />
               </EuiFlexItem>
             )}
-            {isMultiField && (
-              <EuiFlexItem grow={false} className="mappingsEditor__fieldsListItem__icon">
-                <EuiIcon color="subdued" type="link" />
-              </EuiFlexItem>
-            )}
             <EuiFlexItem grow={false} className="mappingsEditor__fieldsListItem__name">
               {source.name}
             </EuiFlexItem>
             <EuiFlexItem grow={false}>
-              <EuiBadge color="hollow">{TYPE_DEFINITION[source.type].label}</EuiBadge>
+              <EuiBadge color="hollow">
+                {TYPE_DEFINITION[source.type].label}
+                {isMultiField ? ' multi-field' : ''} {/* e.g., "Text multi-field" */}
```

Comment:
Can we i18n this? I think we'll want to interpolate the data type too, so the translators can rearrange the words if necessary.

```js
i18n.translate('key', {
  defaultMessage: '{dataType} multi-field',
  values: {
    dataType: TYPE_DEFINITION[source.type].label,
  },
})
```

Reply:
Good catch - fixed.